homeassistant/components/energyzero/quality_scale.yaml

```diff
@@ -0,0 +1,100 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules:
+    status: exempt
+    comment: |
+      It uses a coordinator.py file, but an entity.py file
+      is not used since it only has a sensor platform.
+  config-flow-test-coverage: done
```

Comment:
Optional: I personally dislike snapshot tests in the config flow as the config flow result is concrete data that we want to check, as opposed to snapshotting entities where we have a massive amount of data that we want to fixate

Reply:
Agree on that, will remove it with another PR.